.github/workflows/upload_csv_to_google_cloud.yml

```diff
@@ -101,6 +101,37 @@ jobs:
         destination: '${{ vars.DATA_BUCKET }}/${{ env.FOLDER_NAME }}'
         process_gcloudignore: false
 
+    # Upload agency-specific input CSVs and save URLs
+    - id: 'upload-agency-csvs'
+      uses: 'google-github-actions/upload-cloud-storage@v2'
+      with:
+        path: '${{ vars.DATA_DIR }}/fuse_agency'
```

Comment:
@ayyubibrahimi do you know how we should correctly reference the location of the ephemeral CSV files that were generated in the previous steps of the GitHub Actions?

Reply:
I believe that this right @rajivsinclair